tools/linter/funcorder/cmd/funcorder/main.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"github.com/uber/cadence/tools/linter/funcorder"
+	"golang.org/x/tools/go/analysis/singlechecker"
+)
+
+func main() {
```

Comment:
How is the tool used? Do we need to run it manually? Do we need to pass the file we want to fix?

Reply:
Currently it's a script that runs manually. go run main.go will output the command line.

Reply:
I'm guessing this isn't hooked into the `make lint` step yet?  should that be done?

Reply:
ah.  and we should probably move this to internal/tools/go.mod